From 4536470a6fa248f4c3d0a3ddc60a5249c56787b0 Mon Sep 17 00:00:00 2001
From: Hyungwon Hwang <hyungwon.hwang7@gmail.com>
Date: Sat, 2 Jan 2016 13:49:58 +0900
Subject: [PATCH] Add 4th argument for AC_RUN_IFELSE to tinyxml

AC_RUN_IFELSE compiles and runs a sample test program. But in case of
cross compiling, host machine cannot run the executable. So 4th argument
must be specified.

This patch fixes this error:
configure: error: in `/home/hwh/git/yocto/poky/build_weston/tmp/work/cortexa7hf-vfp-vfpv4-neon-poky-    linux-gnueabi/kodi/15.1-r0/git':
configure: error: cannot run test program while cross compiling
See `config.log' for more details

Signed-off-by: Hyungwon Hwang <hyungwon.hwang7@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index d2cd5b2..df391a0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1164,7 +1164,7 @@ int main() {
   if (TIXML_MAJOR_VERSION < 2) exit(1);
   if (TIXML_MAJOR_VERSION == 2 && ( TIXML_MINOR_VERSION < 6 || ( TIXML_MINOR_VERSION == 6 && TIXML_PATCH_VERSION < 2 ))) exit(1);
 }
-    ]])],[AC_CHECK_LIB([tinyxml], [main],, AC_MSG_ERROR("tinyxml >= 2.6.2 not found"))], [AC_MSG_ERROR([tinyxml >= 2.6.2 not found])]
+    ]])],[AC_CHECK_LIB([tinyxml], [main],, AC_MSG_ERROR("tinyxml >= 2.6.2 not found"))], [AC_MSG_ERROR([tinyxml >= 2.6.2 not found])], [AC_CHECK_LIB([tinyxml], [main],, AC_MSG_ERROR("tinyxml >= 2.6.2 not found"))]
   )])
 AC_LANG_POP([C++])
 
-- 
2.5.0

